composer.json

```diff
@@ -26,11 +26,11 @@
         }
     ],
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.2.0 || ~8.3.0",
         "ext-phar": "*",
         "composer-runtime-api": "^2.0.0",
         "nikic/php-parser": "^4.17.1",
-        "symfony/console": "^6.4.1",
+        "symfony/console": "^7.0.1",
```

Comment:
Is it possible to still allow `symfony/console:^6` to allow others to upgrade? Otherwise we're punching them with a heavy multi-dependency bump 🤔

Reply:
Sure, I updated the PR